article/_posts/2019-07-29-Dont-Worry-About-The-Weather.md

```diff
@@ -0,0 +1,101 @@
+tags: deep-learning CNN segmentation domain-adaptation GAN unsupervised traffic
+cite:
+    authors: "Horia Porav, Tom Bruls, Paul Newman"
+    title:   "Don't Worry About the Weather: Unsupervised Condition-Dependent Domain Adaptation"
+    venue:   "Arxiv, submitted on 25 Jul 2019"
+pdf: "https://arxiv.org/pdf/1907.11004"
+---
+
+# Introduction
+
+This paper underlines the fact that performance degrades quickly when the input conditions change event a little.
+The authors present a domain adaptation system that uses light-weight input adapters to preprocesses input images, irrespective of their appearance, in a way that makes them compatible with *off-the-shelf* computer vision tasks that are trained only on inputs with ideal conditions.
+
+
+The paper shows a hybrid method, where multi-modal data generated in an unsupervised fashion with approximated ground truth. The method is followed by supervised training of domain-adapters, for multiples computer vision tasks, using this generated data and approximated ground truth.
+
+
+This approach has for goal to incrementally adapt to a new, unseen domain.
+If the condition of the input images does not match one that the system has been previously trained on.
+The unsupervised style transfer pipeline will select a model that is the closest to the current condition, clone it, and fine-tune this cloned model to be able to change the style of the reference sequence so that it matches the style of the current input images.
+Those images will be used to train, in a supervised fashion, an additional condition-specific image adapter that will allow upstream computer vision tasks to perform well on the new input image condition.
+
+
+The main contributions:
+
+- Using cycle-consistency GANs to generate multi-condition training data with approximated ground truth for a battery of *off-the-shelf* computer vision tasks.
+
+- Training input image adaptors by using the *off-the-shelf* computer vision models to generate a supervisory signal.
+
+- Enabling online learning of new, unseen domains by leveraging the unsupervised data generation pipeline along with domains on which the data generation models have already been trained.
+
+- Showing that training multiple light-weight adapter modules is better than training monolithic computer vision models that are invariant to input distributions.
+
+
+# Learning Condition-Dependent Representations
+
+![](/article/images/Weather/Gan.png)
+
+![](/article/images/Weather/archi.png)
+
+## Synthetic Multi-Condition Data
+
+They select a reference sequence of images: daytime, clear, and overcast.
+Besides, they join some traversals with severe conditions: night, rain, snow, etc.
+Along with a cycle-consistency architecture, GAN to train generative models that can apply style transfer to the reference condition in order to create a number of synthetic sequences that maintain the structure and geometry of the reference condition.
```

Comment:
_cycle-consistency architecture, GAN _: _cycle-consistency GAN architecture_ ?

Also the sentence does not seem to be finished/it is too long/does not convey a clear idea.

Reply:
Addressed in 9202b1c.